docs/openapi/index.yaml

```diff
@@ -0,0 +1,16 @@
+openapi: 3.0.2
+tags:
+  - name: status
+info:
+  version: 0.0.1
+  title: TTADP
+  contact:
+    email: josh@adhocteam.us
+servers:
+  - url: http://localhost:8080
+  - url: http://backend:8080
+  - url: https://tta-smarthub-sandbox.app.cloud.gov
```

Comment:
Should we also add the other environment servers here (dev, staging, prod) ? Or perhaps incorporate variables in urls?